packages/toolpad-app/src/toolpad/AppEditor/HierarchyExplorer/CreateCodeComponentNodeDialog.tsx

```diff
@@ -73,6 +73,10 @@ export default function CreateCodeComponentDialog({
       <DialogForm
         autoComplete="off"
         onSubmit={(e) => {
+          if (!name || isInvalid) {
+            return;
+          }
+
           e.preventDefault();
```

Comment:
**Off-topic**

No shorthands please
```suggestion
          event.preventDefault();
```

Reply:
I also don't prefer shorthands, but this should be a convention agreed upon with entire team as this seems to be the pattern through the code base.
If we have an agreement to change it, then IMO we should do it across entire codebase

Reply:
> If I understand correctly that's an unwritten convention that we should guard against during PR reviews as well?

Sure, feel free to point it out or fix when you notice it. But I wouldn't make this a bigger thing than it needs to be. IMO It's part of the practice of writing readable, defensive, maintainable, self-documenting code and if that was easily codified in simple rules then there wouldn't be any bad code out there.

Reply:
If we are talking about codified rules, maybe we could give a try with https://eslint.org/docs/latest/rules/id-length 🤔

Reply:
I feel like there are too many exceptions and too little benefit to make this rule worthwhile. e.g. 

loop variables:
```tsx
for (let i = 0; i < rows; i += 1) {
   for (let j = 0; j < columns; j += 1) {
```

math functions:
```tsx
const add = (a, b) => a + b;
```

coordinates:
```tsx
mouse.click(x, y);
```
...